08-Placing-Barriers/03-Wall-geometries.ipynb

```diff
@@ -0,0 +1,223 @@
+{
```

Comment:
<p>Does it mean <em>sphere with inside=False</em> is not periodic across the box, but only cylindrical and planar wall are expected to be periodic(infinite)?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/hoomd-examples/pull/111/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Particles are never outside the primary box image in HOOMD.</p><p> </p><p>Spheres are not infinite surfaces by definition. If you were to place a sphere with a radius larger than box_L/2, then there would be discontinuities for particles near the edge of the box. Consider a radius box_L/2 + 0.1. The particle right on the box edge will still interact with the portion of the sphere just outside the box on the right edge - which would prevent it from ever exiting the primary box through the "gap". </p>